easypost/services/claim_service.py

```diff
@@ -0,0 +1,58 @@
+from typing import (
+    Any,
+    Dict,
+    Optional,
+)
+
+from easypost.easypost_object import convert_to_easypost_object
+from easypost.models import Claim
+from easypost.requestor import (
+    RequestMethod,
+    Requestor,
+)
+from easypost.services.base_service import BaseService
+
+
+class ClaimService(BaseService):
+    def __init__(self, client):
+        self._client = client
+        self._model_class = Claim.__name__
+
+    def create(self, **params) -> Claim:
+        """Create a Claim."""
+        url = "/claims"
+
+        response = Requestor(self._client).request(method=RequestMethod.POST, url=url, params=params, beta=True)
+
+        return convert_to_easypost_object(response=response)
```

Comment:
Can we use `_create_resource` from the parent class?

Reply:
Unfortunately not, because `_create_resource` would wrap the parameters in a top-level key, therefore sending the wrong payload schema.

Reply:
Gotcha. I will approve when endpoint is GA